sklearnex/dispatcher.py

```diff
@@ -45,6 +51,15 @@ def get_patch_map():
         mapping['svc'] = [[(svm_module, 'SVC', SVC_sklearnex), None]]
         mapping['nusvr'] = [[(svm_module, 'NuSVR', NuSVR_sklearnex), None]]
         mapping['nusvc'] = [[(svm_module, 'NuSVC', NuSVC_sklearnex), None]]
+        mapping['set_config'] = [[(base_module,

```

Comment:
When one or several algorithms are patched, the config will not be patched. Configs will only be patched in two versions:
1. `patch_sklearn()`
2. `patch_sklearn(["set_config", "get_config", "config_context"])`

I think we need to patch these functions in other cases.